### PR TITLE
Added navbar

### PR DESCRIPTION
**Pull request by vinaybajjuri58**
Added Navbar

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        ><span><i class="fas fa-bars"></i></span
+      ></label>
+      <nav class="navbar">
+        <ul class="nav-items">
+          <li><a href="#">Home</a></li>
+          <li><a href="#">About</a></li>
+          <li><a href="#">Blog</a></li>
+          <li><a href="#">Contact</a></li>
+        </ul>
+      </nav>
```
Comment (vikasvk11) [inline index.html:29]: Clean code! Nice!

File: index.html [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script
+      src="https://kit.fontawesome.com/b0efec786b.js"
+      crossorigin="anonymous"
+    ></script>
+    <!-- <link rel="stylesheet" href="style.css" /> -->
```
Comment (khusharth) [inline index.html]: Can remove the commented code!

**Review by khusharth — CHANGES_REQUESTED** [review]

File: index.html (hunk 2 above)
Comment (vinaybajjuri58) [reply index.html]: Okay ill do that

Comment (vinaybajjuri58) [reply index.html]: > 
> 
> Can remove the commented code!

Hey Khusharth I have removed the commented code 
Can you please check that.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+      <label class="nav-toggle-label" for="nav-toggle"
+        ><span><i class="fas fa-bars"></i></span
+      ></label>
+      <nav class="navbar">
+        <ul class="nav-items">
+          <li><a href="#">Home</a></li>
+          <li><a href="#">About</a></li>
+          <li><a href="#">Blog</a></li>
+          <li><a href="#">Contact</a></li>
+        </ul>
```
Comment (aneenajohn) [inline index.html:28]: Nav bar looks good to go!

**Review by aneenajohn — COMMENTED** [review]
Looks good to be merged!

File: index.html (hunk 3 above)
Comment (Aravindan07) [inline index.html:28]: Hey Vinay, your navbar code is easily understandable and well-written follow the same in other projects also.

**Review by Aravindan07 — COMMENTED** [review]
Perfect to be merged!

File: index.html (hunk 3 above)
Comment (vinaybajjuri58) [reply index.html:28]: Yeah okay Aravindan

Comment (vinaybajjuri58) [reply index.html:28]: Thank you aneena

File: index.html [hunk 4]
```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script
+      src="https://kit.fontawesome.com/b0efec786b.js"
+      crossorigin="anonymous"
```
Comment (sjsouvik) [inline index.html:9]: Great work Vinay!

**Review by sjsouvik — COMMENTED** [review]
Go ahead and merge it

File: index.html (hunk 4 above)
Comment (vinaybajjuri58) [reply index.html:9]: Thank you

**Pull request merged**